src/components/views/elements/AppTile.tsx

```diff
@@ -164,26 +165,43 @@ export default class AppTile extends React.Component<IProps, IState> {
     };
 
     private onWidgetLayoutChange = () => {
-        const room = MatrixClientPeg.get().getRoom(this.props.room.roomId);
-        const app = this.props.app;
-        const isActiveWidget = ActiveWidgetStore.instance.getWidgetPersistence(app.id);
-        const isVisibleOnScreen = WidgetLayoutStore.instance.isVisibleOnScreen(room, app.id);
+        const room = this.context.getRoom(this.props.room.roomId);
```

Comment:
I'm quite confused by this line?
Why do we need to get the room from the Matrix client if it's already been passed down as props? It looks like I'm missing some bits here

Reply:
True, think this was copy-pasted from elsewhere that didn't get the room